src/reporters/jsonReporter.ts

```diff
@@ -112,7 +112,7 @@ export class JsonReporter {
         value = `${value} ms`;
       }
 
-     return Object.assign(summary, { [key]: value });
+      return Object.assign(summary, { [key]: value });
```

Comment:
the types on this are quite dishonest.  I get that this was inherited from the apex folks, but the keys on `Summary` get mangled by the `time` stuff and don't reflect what's actually there.